docs/projects/fps/overview.md

```diff
@@ -1,11 +1,15 @@
-# FPS Starter Project
+# FPS Starter Project: Overview
 
 <img src="{{assetRoot}}assets/fps/enemies.png" style="margin: 0 auto; width: 100%; display: block;" />
 
-The FPS starter project is a great starting point for working on the SpatialOS GDK for Unity. Many projects, whatever their genre, share some core elements and this starter project can get you off to a easy start.
+The FPS (First-person Shooter) Starter Project is a great starting point for working with the SpatialOS GDK for Unity. Many projects, whatever their genre, share some core elements and this starter project can get you off to a good start.
 
-The SpatialOS GDK can help you run your games at scale, making it easy for even a small team to make a massive multiplayer hit game. The FPS starter project is already configured to scale-up, and you can test it out right now! Check out the [FPS at scale]({{urlRoot}}/content/get-started/get-started) guide to dive right in!
+The SpatialOS GDK can help you run your games at scale, making it easy for even a small team to make a massive multiplayer hit game. The FPS Starter Project is already configured to scale-up, so you can test SpatialOS out right now. 
+
+To dive right in, follow the [Get started]({{urlRoot}}/content/get-started/get-started) guide which uses the FPS Starter Project. Once you have completed this, you can follow the _Add your first feature_ guide which is a Health Pick-up tutorial, following on from the _Get started_ guide.
```

Comment:
Should the "Add your first feature" guide be a link?

Reply:
Not at this point, as we don't want anyone to go there at this point - it's only useful once you have followed to _Get started_ guide which uses the FPS Starter Project.